dotnet-testing/EFCoreMocking/Tests/Tests/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
Let's use this '`Usings.cs` class. You have a couple of same `using` statements in different test classes. For example, `using EFCoreMocking.Data`, `using Moq`, etc.

Let's move all of these using statements that are used in different test class to this one, and save some lines :)